From 9b523e1fc8a786da6fb6e0a8301d2fb4a2332dbb Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 10 Mar 2023 15:07:10 +0000
Subject: [PATCH 616/726] overlays: vc4-kms/fkms-v3d: Raise CMA to 512MB

On Pi 4, raise the requested CMA size to 512MB by default, on the
understanding that the firmware will cap that to 256MB if it's a
32-bit kernel with less than 2GB of RAM.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/vc4-fkms-v3d-pi4-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-pi4-overlay.dts b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-pi4-overlay.dts
index 7792ead0cbb3..0271d90d6a85 100644
--- a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-pi4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-pi4-overlay.dts
@@ -8,7 +8,7 @@
 #include "cma-overlay.dts"
 
 &frag0 {
-	size = <((320-4)*1024*1024)>;
+	size = <((512-4)*1024*1024)>;
 };
 
 / {
diff --git a/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts
index 1cf4ec5c8637..c1e53e3ed575 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts
@@ -10,7 +10,7 @@
 #include "cma-overlay.dts"
 
 &frag0 {
-	size = <((320-4)*1024*1024)>;
+	size = <((512-4)*1024*1024)>;
 };
 
 / {
-- 
2.33.1

